### ISSUE TITLE
`oc debug node` not possible in okd without valid token

### ISSUE THREAD

**Issue by gtema** [openshift/okd#57]
When installing OKD (4.4) without pullSecret `oc debug node` is not working, since registry.redhat.io/rhel7/support-tools can not be pulled.

```
Failed to pull image "registry.redhat.io/rhel7/support-tools": rpc error: code = Unknown desc = unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
```

**Comment by vrutkovs** [comment]
Workaround: `oc debug --image=docker.io/centos/tools node/...`. 

Not sure if it can use that image out of the box - iiuc the name is hardcoded in `oc`. @soltysh?

**Comment by soltysh** [comment]
> Not sure if it can use that image out of the box - iiuc the name is hardcoded in `oc`. @soltysh?

No, we can't hardcoded it, yet.

**Comment by vrutkovs** [comment]
This is fixed in 4.5 - https://github.com/openshift/oc/pull/368. oc would now use `tools` imagestream, included in the payload.

That would be resolved in 4.5-based build of OKD

**Comment by vrutkovs** [comment]
Fixed in 4.5 nightlies (and upcoming Beta 6). `oc debug` uses `tools` image from the payload. Make sure you're using oc 4.5 for that
